tests/admin_filters/tests.py

```diff
@@ -154,6 +154,30 @@ def expected_parameters(self):
         return [self.lookup_kwarg]
 
 
+class DepartmentOwnershipListFilter(SimpleListFilter):
+    title = "Department Ownership"
+    parameter_name = "department_ownership"
+
+    def lookups(self, request, model_admin):
+        return [
+            ("OWNED", "Owned by Department"),
+            ("NOT_OWNED", "Not Owned by Department"),
+        ]
+
+    def queryset(self, request, queryset):
+        queryset = queryset.annotate(
+            owned_book_count=models.Count(
+                "employee__department",
+                filter=models.Q(employee__department__isnull=False),
```

Comment:
Department is non-nullable so we should use something else, e.g. `__code`.
```suggestion
                filter=models.Q(employee__department__isnull=False),
```